src/FirebaseProjectManager.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Laravel\Firebase;
+
+use Illuminate\Contracts\Foundation\Application;
+use Kreait\Firebase\Factory;
+use Kreait\Firebase\Exception\InvalidArgumentException;
+
+class FirebaseProjectManager
+{
+    /** @var \Illuminate\Contracts\Foundation\Application */
```

Comment:
Could you check if this equally works with `Illuminate\Contracts\Container\Container`? This would probably mean that getting the config would have to be implemented differently, but I remember changing `Application` to `Container` in the past for better Lumen support.

Reply:
I'll have a look into it. I changed it to copy the `resolveCredentials` method to `FirebaseProjectManager`, as `basePath` does not exist on `Container`.